blockchain/types/transaction.go

```diff
@@ -78,7 +78,7 @@ type Transaction struct {
 	markedUnexecutable int32
 
 	// lock for protecting AsMessageWithAccountKeyPicker().
-	mu sync.Mutex
+	mu sync.RWMutex
```

Comment:
Why did you change the type of mutex? Because we only use mutex in case of reading?

Reply:
@hqjang-pepper No, this is because the mutex is used for reading and writing.